pulsar/consumer.go

```diff
@@ -231,6 +231,14 @@ type Consumer interface {
 	// AckID the consumption of a single message, identified by its MessageID
 	AckID(MessageID) error
 
+	// CumulativeAck the reception of all the messages in the stream up to (and including)
+	// the provided message.
+	CumulativeAck(msg Message) error
+
+	// CumulativeAckID the reception of all the messages in the stream up to (and including)
+	// the provided message, identified by its MessageID
+	CumulativeAckID(msgID MessageID) error
```

Comment:
Maybe it's better to use `AckIDCumulative` as the method name? Like the `acknowledge` and `acknowledgeCumulative` methods in other clients.

Reply:
Good suggestion. I will rename the two methods.